### DISCUSSION TITLE
Charging

### DISCUSSION THREAD

**Discussion by dixkxnsks** [ianjwhite99/sync-connect-sdk#62] [Ideas]
Could you add charging control (start, stop)?

**Comment by ianjwhite99** [comment]
> Could you add charging control (start, stop)?

Hi,

I've gotten a few messages about this. I will try and get this added sometime in the next few days.

**Reply by dixkxnsks** [reply]
Hi, any progress?

**Comment by dixkxnsks** [comment]
Thanks a lot

**Comment by wolfgangB33r** [comment]
Yes charging control would be super cool. Thanks by the way for this absolutely amazing piece of code!
